blocks/autocompleters/index.js

```diff
@@ -122,6 +122,10 @@ export function userAutocompleter() {
 		return textNode.parentElement.closest( 'a' ) === null;
 	};
 
+	const setSearch = ( search, getOptions ) => {
```

Comment:
Why is `getOptions` a parameter? Why not just pass the search?

Reply:
I am calling getOptions in the inner function, I couldn't seem to call it without passing, I'll double check this

Reply:
removed setSearch entirely so this no longer applies